commercial/app/controllers/HostedContentController.scala

```diff
@@ -84,13 +84,13 @@ class HostedContentController(contentApiClient: ContentApiClient)
           val itemId = s"advertiser-content/$campaignName/$pageName"
           contentType match {
             case "video" =>
-              val trails = HostedTrails.fromContent(itemId, trailCount = 1, results)
+              val trails = HostedTrails.fromContent(itemId, trailCount = items.getOrElse(1), results)
               Cached(cacheDuration)(JsonComponent(hostedVideoOnward(trails.headOption)))
             case "article" =>
-              val trails = HostedTrails.fromContent(itemId, trailCount = 2, results)
+              val trails = HostedTrails.fromContent(itemId, trailCount = items.getOrElse(2), results)
```

Comment:
And then building hosted trails becomes a lot simpler so could be done in a different def, without specifying a trail count at all.  But that could be done in a separate PR.